packages/terra-consumer-nav/src/components/nav-items/NavItem.scss

```diff
@@ -2,7 +2,7 @@
 
 :local {
   .item {
-    color: var(--terra-consumer-nav-item-color, #fff);
+    color: var(--terra-consumer-item-color, #fff);
```

Comment:
Should the variables in this file keep the `nav` word in the variable name? They all look specific to nav.